### ISSUE TITLE
Update reflection

### ISSUE THREAD

**Issue by katieb1** [UBC-MDS/DSCI532_Group204_Vino#83]
We need to update the reflection to incorporate the following:

- [x]  Start with the reflection.md file you included with milestone2

- [ ]  Make any necessary edits or changes (for e.g., if you indicated in your milestone2 reflection that your app had a bug and you fixed it in week 3, you should update your reflections file)

- [ ] Wishlist features/bug-fixes

- [ ] If there is anything you wish you had more time to implement or fix, you can list it here.

- [ ] Include a brief, high-level summary of any changes you decided to make

- [ ] Provide a brief, high-level summary of the feedback you received during the lab session 

Examples of questions we can answer for this part are as follows:
* Reflect on the usability of your app. How easy or hard was it for your peers to use your app? Did you expect them to use the app in they way they did?
* What things did you hear that's similar across reviewers? Is there a theme here?
* From the feedback, what things do you think are appropriate to change in your app, particularly given the time frame you have to improve the app?
* What is unreasonable to change, and why? Time restraints? Too technical?
* What feedback was the most/least valuable?
* What part of the feedback process was most/least valuable?
* What did you learn from your experience being a "fly-on-the-wall". Was this useful? Why or why not?
* Did the feedback process lead to an improved app? What parts were valuable? What parts were not valuable?

- [ ] Reflect on the feedback you received (more guidance is below)

**Comment by brendoncampbell** [comment]
The feedback summary in issue #73 should be a good starting point for a lot of the prompt questions above. 

Some feedback items that I tried to address:
- Clearer plot titles
- Updated plot axis title to include units
- Updated description text within the app to clarify our calculated wine 'value' score
- Fixed axis label angling on heatmap (thanks Katie!)

Jarvis has correctly noted that due to the circumstance and noise in the lab room, our peers weren't really able to use and absorb the functionality of the app in detail. Some feedback items can likely be attributed to this circumstance, and may not need to be actioned

**Comment by jnederlo** [comment]
**Feedback:**
- Not entirely sure what to do with the app.
- Maybe a little bit too wordy.
- Not enough guidance (directly conflicting with above).
- Aesthetically pleasing app and design, consistent color scheme.
- Good interactivity.

**Feedback Reflection:**
- Most of the feedback we received gave me very little value. The room was too noisy and cluttered and most people didn't read the very short description describing what to do with our app. Their feedback regarding ease of use reflected this and as such is not considered valid feedback to me.
- We did get very useful feedback regarding what features users liked and the overall design. Our app had a strong layout and speaks to why we received that feedback.
- Some feedback that was useful, which I think we were already aware of, was regarding additional features. Users wanted the map to show other details like the average points. This was not possible in our map because there wasn't a strong representation from all states, so numbers would have been exaggerated. This is, however, possible with a clickable interactive addition to the map, which would produce new charts based on which state was clicked on, thereby showing the information not on the map, but with an associated chart. This feature is a large addition, and will be introduced in a later version, but is not something appropriate for these maintenance items.
- Additional feedback requested the ability to display particular bottles of wine. I would consider this to be very encouraging, as this is exactly what we wanted to get from users. They liked our app so much that they wanted to use it to make future decisions. Adding in a search function for the database to display results is another large undertaking and is planned to be released in a future version. Like the above, it is not something appropriate for these maintenance items.

**Maintenance Strategy:**
- No real known bug fixes needed.
- Make aesthetic and simple UI changes based on user feedback.
- Leave additional features for future releases.
- Refactor codebase for readability in PEP-8 format.
- Re-organize file tree to better organizational structure.

**Maintenance Performed:**
- We implemented descriptive UI changes to some plot titles, and made charts consistent regarding points vs rating. All charts now refer to the review's 'Points' field as 'Rating'.
- Re-factored codebase to clean up and make more readable. Formatted to PEP-8 style guide.
- Re-structured file tree to move source code files to src, and made separate theme module to reference plot themes.

**Reflection:**
I am very happy with how version 1 of our app turned out. I know there are more features to add, and I plan to add some, but we did a very good job making an interactive dashboard that could provide the user with real value. Additionally, the feedback didn't find any bugs or major issues which speaks to the quality of what we produced in version 1. Our effort put into making the app aesthetically pleasing was noticed by users, and doesn't need much improvement, if any. I think we did an excellent job of addressing our initial research questions.

**Comment by katieb1** [comment]
Alright, I have incorporated your discussion points and added my own points as well into our new reflection. Please take a look at the pull request and let me know if we should add/change/delete anything.
